### PR TITLE
kubevirt: vm-console-proxy: Enable CI for release branches.

### PR DESCRIPTION
**Pull request by akrejcir**
Updated `build_root`, and enabled CI for branches:
- `release-v0.2`
- `release-v0.3`

### CODE REVIEW

File: ci-operator/config/kubevirt/vm-console-proxy/kubevirt-vm-console-proxy-release-v0.2.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unittests
+  skip_if_only_changed: ^\.github/|^docs/|^example-client/|\.[mM][dD]$|^.gitignore$|^OWNERS$|^LICENSE$
+  steps:
```
Comment (ksimon1) [inline ci-operator/config/kubevirt/vm-console-proxy/kubevirt-vm-console-proxy-release-v0.2.yaml:29]: please specify here cluster_profile: gcp-virtualization

File: ci-operator/config/kubevirt/vm-console-proxy/kubevirt-vm-console-proxy-release-v0.3.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unittests
+  skip_if_only_changed: ^\.github/|^docs/|^example-client/|\.[mM][dD]$|^.gitignore$|^OWNERS$|^LICENSE$
+  steps:
+    test:
```
Comment (ksimon1) [inline ci-operator/config/kubevirt/vm-console-proxy/kubevirt-vm-console-proxy-release-v0.3.yaml:30]: same here

File: ci-operator/config/kubevirt/vm-console-proxy/kubevirt-vm-console-proxy-release-v0.2.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: rhel-8-release-golang-1.19-openshift-4.12
```
Comment (ksimon1) [inline ci-operator/config/kubevirt/vm-console-proxy/kubevirt-vm-console-proxy-release-v0.2.yaml:5]: this tag is correct?

File: ci-operator/config/kubevirt/vm-console-proxy/kubevirt-vm-console-proxy-release-v0.3.yaml [hunk 4]
```diff
@@ -0,0 +1,62 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: rhel-8-release-golang-1.19-openshift-4.12
```
Comment (ksimon1) [inline ci-operator/config/kubevirt/vm-console-proxy/kubevirt-vm-console-proxy-release-v0.3.yaml:5]: weren't 4.13 based on rhel9?

Comment (akrejcir) [reply ci-operator/config/kubevirt/vm-console-proxy/kubevirt-vm-console-proxy-release-v0.3.yaml]: updated

File: ci-operator/config/kubevirt/vm-console-proxy/kubevirt-vm-console-proxy-release-v0.2.yaml (hunk 1 above)
Comment (akrejcir) [reply ci-operator/config/kubevirt/vm-console-proxy/kubevirt-vm-console-proxy-release-v0.2.yaml:29]: done

File: ci-operator/config/kubevirt/vm-console-proxy/kubevirt-vm-console-proxy-release-v0.3.yaml (hunk 2 above)
Comment (akrejcir) [reply ci-operator/config/kubevirt/vm-console-proxy/kubevirt-vm-console-proxy-release-v0.3.yaml:31]: done

File: ci-operator/config/kubevirt/vm-console-proxy/kubevirt-vm-console-proxy-release-v0.2.yaml (hunk 3 above)
Comment (akrejcir) [reply ci-operator/config/kubevirt/vm-console-proxy/kubevirt-vm-console-proxy-release-v0.2.yaml]: done

File: ci-operator/config/kubevirt/vm-console-proxy/kubevirt-vm-console-proxy-release-v0.2.yaml [hunk 5]
```diff
@@ -0,0 +1,63 @@
+build_root:
```
Comment (ksimon1) [inline ci-operator/config/kubevirt/vm-console-proxy/kubevirt-vm-console-proxy-release-v0.2.yaml:1]: you will have to change this to
```
build_root:
  image_stream_tag:
    name: builder
    namespace: ocp
    tag: rhel-9-golang-1.19-openshift-4.13
```

Comment (akrejcir) [reply ci-operator/config/kubevirt/vm-console-proxy/kubevirt-vm-console-proxy-release-v0.2.yaml:1]: done

**Comment by 0xFelix** [comment]
Looks like `jq` is missing:

```./automation/e2e-functests/versions.sh: line 49: jq: command not found```

**Comment by akrejcir** [comment]
@ksimon1 For now I will not update the image. I will post a PR to the `vm-console-proxy` to include a builder image.

**Comment by ksimon1** [comment]
> @ksimon1 For now I will not update the image. I will post a PR to the `vm-console-proxy` to include a builder image.

ok

**Review by 0xFelix — APPROVED** [review]
/lgtm

Thanks!

**Pull request merged**